docs/update_docs.sh

```diff
@@ -15,8 +15,10 @@ pushd "${DOCS_WORKSPACE}" &> /dev/null
 bazel clean \
 && bazel build //... \
 && cp bazel-bin/*.md . \
-&& chmod 0644 *.md \
-&& git add *.md \
-&& git commit -m "Regenerate documentation"
-popd &> /dev/null
+&& chmod 0644 *.md
+
+if [[ -z "${SKIP_COMMIT:-}" ]]; then
```

Comment:
FTR, I'd be fine with this script not doing the commit at all. In fact I'd slightly prefer not committing.

Reply:
That should be handled by https://github.com/bazelbuild/rules_rust/pull/695 and not this PR. I just want this to not create a separate commit but the changes here would still be added to a commit when the release PR is made.